client/serviceReducer.js

```diff
@@ -64,9 +61,12 @@ function serviceReducer(state, action) {
         filters: action.payload,
       };
     case FETCH_ORDERS_FULFILLED:
+    case POST_ORDER_FULFILLED:
```

Comment:
@IgorChaikin Currently you have Redux actions, which is defined in `client/actions` directory. But all reducers for actions are described here in `client/serviceReducer.js`. Could you restructure component a little to use single file for `action types`, `actions` and `reducers`. Something like this:

<img width="465" alt="image" src="https://user-images.githubusercontent.com/12561302/129892562-6db9d91c-df8b-43ae-a059-eb2e462f90e8.png">